platforms/cuda/src/CudaTorchKernels.cpp

```diff
@@ -35,22 +35,23 @@
 #include "openmm/internal/ContextImpl.h"
 #include <map>
 #include <cuda_runtime_api.h>
-
+#include <c10/cuda/CUDAGuard.h>
+#include <c10/cuda/CUDAStream.h>
 using namespace TorchPlugin;
 using namespace OpenMM;
 using namespace std;
 
 // macro for checking the result of synchronization operation on CUDA
 // copied from `openmm/platforms/cuda/src/CudaParallelKernels.cpp`
-#define CHECK_RESULT(result, prefix) \
-if (result != CUDA_SUCCESS) { \
-    std::stringstream m; \
-    m<<prefix<<": "<<cu.getErrorString(result)<<" ("<<result<<")"<<" at "<<__FILE__<<":"<<__LINE__; \
-    throw OpenMMException(m.str());\
-}
+#define CHECK_RESULT(result, prefix)                                                                                                                                                                   \
+    if (result != CUDA_SUCCESS) {                                                                                                                                                                      \
+        std::stringstream m;                                                                                                                                                                           \
+        m << prefix << ": " << cu.getErrorString(result) << " (" << result << ")"                                                                                                                      \
+          << " at " << __FILE__ << ":" << __LINE__;                                                                                                                                                    \
+        throw OpenMMException(m.str());                                                                                                                                                                \
+    }
```

Comment:
Why did you add all the blank space?  This is likely to confuse anyone looking at the source code, since the `\` characters may be hidden past the edge of the window.